examples/snap-plugin-publisher-file/file/file.go

```diff
@@ -49,6 +50,9 @@ func (f FPublisher) Publish(mts []plugin.Metric, cfg plugin.Config) error {
 	if err != nil {
 		return err
 	}
+	if val, err := cfg.GetBool("return_error"); err == nil && val {
+		return errors.New("Houston we have a problem")
```

Comment:
`<joke>` Not consistent with "Houston, we have a problem!" in the rand collector `</joke>`